### DISCUSSION TITLE
Allow usage domains with prefixes simultaneously

### DISCUSSION THREAD

**Discussion by acibilous** [nuxt-modules/i18n#2083] [Ideas]
### Describe the feature

I need it on my live site support domains and prefixes combination.

domain.uk - UK site with English
domain.be - Belgium site with NL language
domain.be/fr - Belgium site with FR language

Is any ability to configure it and plans to implement it?
If yes please explain how to configure it
Thanks!

### Additional information

- [X] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution) (The contribution guideline of nuxt-modules/i18n is compliant with Nuxt too).
- [X] Check existing [discussions](https://github.com/nuxt-modules/i18n/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by kazupon** [comment]
I ssem that this is feature request, so we will try to move github discussion

**Comment by acibilous** [comment]
any updates guys?
